src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -342,4 +343,47 @@ public void dropDatabase(String databaseName) {
         MongoDatabase db = mongoClient.getDatabase(databaseName);
         db.drop();
     }
+    
+    /**
+     * Check if the document exists
+     * @param databaseName
+     * @param collectionName
+     * @param condition      a condition to find a requested object in the database
+     * @return
+     */
+    public boolean checkObjectExists(String databaseName, String collectionName, String condition) {
```

Comment:
I would prefer to use document in this method name, to be consistent. When we refer to objects, it should be the aggregated objects and other objects we store inside the documents in the database.

Reply:
changed the method name to "checkDocumentExists"